src/ResourceManager/DataFactories/Commands.DataFactoryV2.Test/ScenarioTests/IntegrationRuntimeTests.ps1

```diff
@@ -112,6 +112,7 @@ function Test-SsisAzure-IntegrationRuntime
             -CatalogAdminCredential $mycreds `
             -CatalogPricingTier 'S1' `
             -MaxParallelExecutionsPerNode 1 `
+            -LicenseType LicenseIncluded `
```

Comment:
@HaoQian-MS it doesn't look like any of the recordings were updated for this test, so the recorded requests and responses won't include this parameter

Reply:
Checked again. This ps function 'Test-SsisAzure-IntegrationRuntime' is not included in test (IntegrationRuntimeTests.cs) yet. So, there's no original recordings for me to update.